experimental/generated/beats/fields.ecs.yml

```diff
@@ -9892,10 +9848,10 @@
       type: keyword
       ignore_above: 1024
       description: "Type of indicator as represented by Cyber Observable in STIX 2.0.\n\
-        Expected values\n  * autonomous-system\n  * artifact\n  * directory\n  * domain-name\n\
-        \  * email-addr\n  * file\n  * ipv4-addr\n  * ipv6-addr\n  * mac-addr\n  *\
-        \ mutex\n  * process\n  * software\n  * url\n  * user-account\n  * windows-registry-key\n\
-        \  * x-509-certificate"
+        Recommended values:\n  * autonomous-system\n  * artifact\n  * directory\n\
```

Comment:
Some (but not all) Newlines appear to have been saved as "\n" is this intentional?

Reply:
I believe the YAML loading/dumping uses an 80 character line length, but then we have our own formatting introducing newlines to get the field description to render nicely in the docs:

https://ecs_1471.docs-preview.app.elstc.co/guide/en/ecs/master/ecs-threat.html#field-threat-indicator-confidence

Reply:
Thanks for the info, Eric.

I'll resolve this conversation, but we can reopen it if needed.